frontend/e2e_frontend.containerfile

```diff
@@ -1,4 +1,4 @@
-FROM cypress/browsers:22.14.0
+FROM docker.io/cypress/browsers:22.14.0
```

Comment:
This is actually interesting ... although the container files seemed to work just fine in the CI, whenever I run functional tests locally (backend or frontend) I got a podman *prompt* to choose one of three repos (docker, fedora, and redhat), which is annoying. It seems that the container image isn't actually available from either of the others, and specifying `docker.io/` here avoids the prompt. (Even though I'd rather not pull from `docker.io` as our access is throttled.) If you have any better ideas, I'd love to hear them.